app/views/communities/index.js.erb

```diff
@@ -4,4 +4,14 @@ $("<%=j render partial: "list"%>").appendTo($("#my-communities"));
   $('#infinite-scrolling').remove();
 <% else %>
   $('#infinite-scrolling').html("<%=j link_to_next_page(@communities, 'See more', :remote => true) %>");
-<% end %>
\ No newline at end of file
+<% end %>
+
+$(window).scroll(function() {
+  var next_url = $("#infinite-scrolling a[rel='next']").attr('href');
+  if (( next_url ) && ((window.innerHeight + window.pageYOffset) >= document.body.offsetHeight))
+  {
+    $('#my-communities').show();
+    $.getScript(next_url);
+    return;
+  }
+});
```

Comment:
is this index.js.erb file really necessary?
can we move this to application.js as a file so it gets loaded the first time when the site loads?

Reply:
we have created infinite scrolling function to application.js